tests/testimonialCarousel.test.ts

```diff
@@ -0,0 +1,33 @@
+describe('Testimonial - Carousel', () => {
+    it('Check the testimonial carousel', async() => {
+        await browser.url('https://staging-new.a2odev.com/')
```

Comment:
Is more organized if all the selectors are declared at the beginning and then be reused at any time that is necessary something like this

        //Selectors
        const webAppsLink = await $('h3=Web Apps Development')
        const webSiteLink = await $('.elementor-icon-box-title=Web Site Development')
        ....
        ....
        
        ///Services Links

        webAppsLink.click()
        await browser.pause(3000)
        expect($('h2=Web App Development')).toBeExisting
        await browser.url('https://staging-new.a2odev.com')
        await browser.pause(3000)

        webSiteLink.click()
        await browser.pause(3000)
        expect($('h2=Website Development')).toBeExisting
        await browser.url('https://staging-new.a2odev.com')
        await browser.pause(3000)